plugins/GSdx/GSSetting.cpp

```diff
@@ -174,6 +174,9 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "Uploads GS data when rendering a new frame to reproduce some effects accurately. Fixes black screen issues in games like Armored Core: Last Raven.";
 		case IDC_MIPMAP_HW:
 			return	"Control the accuracy level of the mipmapping emulation\n\n"
+				"Automatic:\nAutomatically sets the recommended mipmapping level based on games that need mipmapping.\n"
+				"This is the recommended setting.\n"
+				"Off will be selected for games that don't need mipmapping support.\nBasic will be selected for games that need Basic mipmapping support.\nFull will be selected for games that need Full mipmapping support.\n\n"
```

Comment:
Line 178 and 179 seems sort of redundant to me.

You've already hinted at 177 that this option sets the recommended mipmapping level, so it'd obviously be the recommended setting.

And Line 179, I'm not too fussed about this but it's just saying the same thing as the combobox texts, ``Basic will be selected for games that need Basic mipmapping support`` and ``Full will be selected for games that need Full mipmapping support``. Though if you do want to keep it, please remove capitalization in the second instance of ``Basic`` and ``Full`` you had used in your sentences.